package.json

```diff
@@ -87,7 +87,17 @@
   },
   "jupyterlab": {
     "extension": true,
-    "outputDir": "jupyterlab_niryo_one/labextension"
+    "outputDir": "jupyterlab_niryo_one/labextension",
+    "sharedPackages": {
+      "jupyterlab-blockly": { 
+        "bundled": false, 
+        "singleton": true 
+      }, 
+      "blockly": { 
+        "bundled": false, 
+        "singleton": true 
+      } 
+    }
```

Comment:
@DenisaCG We need to revert this change. I believe this is the reason why we are seeing the error `Uncaught Error: Language "Python" does not know how to generate code for block type "niryo_one_connect".`.

This configuration should be present only in the extension that export the plugin.

Reply:
Line 90 it's ok. We need to remove the `sharedPackages` attribute.

Reply:
@hbcarlos Isn't this a case of requiring a service and as such following under the `sharedPackages` configuration detailed [here in the documentation](https://jupyterlab.readthedocs.io/en/stable/extension/extension_dev.html#requiring-a-service)?